### PR TITLE
Add sample warnings in transformation

### PR DESCRIPTION
**Pull request by rkondratowicz**

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/transformation/services/CaseValidator.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.bulkscanccdeventhandler.transformation.services;
+
+import com.google.common.base.Strings;
```
Comment (doncem) [inline src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/transformation/services/CaseValidator.java]: Either guava's or spring's `StringUtil`. Just a note ;)

File: src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/transformation/services/CaseValidator.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+@Component
+public class CaseValidator {
+
+    public List<String> getWarnings(SampleCase theCase) {
+        return Strings.isNullOrEmpty(theCase.email)
+            ? singletonList("'email' is empty")
+            : emptyList();
+    }
+}
```
Comment (doncem) [inline src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/transformation/services/CaseValidator.java]: Looks like a good candidate for utility object does it not?

Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/transformation/services/CaseValidator.java]: A proper compenent IMHO.
I inject this one to transformer , and it's quite useful to be able to mock the responses.
Exception record validator is not static either.

File: src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/transformation/services/CaseValidator.java (hunk 1 above)
Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/transformation/services/CaseValidator.java]: this is guava

Comment (doncem) [reply src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/transformation/services/CaseValidator.java]: yes it is. like i said - just a note. if possible to achieve everything with one library - there's no need of extra one just to do some "nice to have" things if it's appropriate to call like that. once again - just a note. either way it's good

File: src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/transformation/services/CaseValidator.java (hunk 2 above)
Comment (doncem) [reply src/main/java/uk/gov/hmcts/reform/bulkscanccdeventhandler/transformation/services/CaseValidator.java]: Why would you want to mock that?

**Pull request merged**